pkgs/development/libraries/at-spi2-atk/default.nix

```diff
@@ -42,7 +42,7 @@ stdenv.mkDerivation rec {
     description = "D-Bus bridge for Assistive Technology Service Provider Interface (AT-SPI) and Accessibility Toolkit (ATK)";
     homepage = https://gitlab.gnome.org/GNOME/at-spi2-atk;
     license = licenses.lgpl2Plus; # NOTE: 2018-06-06: Please check the license when upstream sorts-out licensing: https://gitlab.gnome.org/GNOME/at-spi2-atk/issues/2
-    maintainers = with maintainers; [ jtojnar gnome3.maintainers ];
+    maintainers = with maintainers; [ jtojnar ] ++ gnome3.maintainers;
```

Comment:
jtojnar is already a member of the gnome3 maintainers list

Reply:
Do you want me to remove them?
It's possible that @jtojnar leaves the gnome3 maintainer list without intending to give up maintenance of at-spi2-atk.

Reply:
:rofl: I'd think jtojnar would have given up on NixOS if it reached that point.

Nothing strange should happen here if they've in this list twice?